app/templates/components/task-card.hbs

```diff
@@ -19,15 +19,24 @@
     <span class="task-card__time" data-test-task-time>{{moment-from-now task.createdAt}}</span>
     <span class="task-card__integrations" data-test-task-integrations>
       {{#if task.githubIssue}}
-        {{github/issue-link
-          githubIssue=task.githubIssue
-          githubRepo=task.githubRepo
-          size="small"
-        }}
+        {{#if hasHovered}}
+          {{github/issue-link
+            githubIssue=task.githubIssue
+            githubRepo=task.githubRepo
+            size="small"
+          }}
+        {{else}}
+          <span class="sprite-icon">
+            <svg class="solid-light-gray size-16">
+              <use xlink:href="#github-48"></use>
+            </svg>
+          </span>
+        {{/if}}
       {{/if}}
     </span>
   </p>
   {{task-assignment
+    hideDropdown=(not hasHovered)
```

Comment:
Would call this something like `renderDropdown` or `dropdownReady` or something to avoid a potentially confusing `not`.

Reply:
Maybe `dropdownVisible`?

Reply:
actually, I'm thinking leave the `not` and call `hideDropdown` something like `dropdownSupressed`. I think it shows intent a bit more, maybe

Possibly even something like `deferredRendering` would maybe work a bit better.